submissions/aymkin/OOP-frogger/app.js

```diff
@@ -0,0 +1,121 @@
+let scoreCount = 0;
+
+const START = {
+	x: 202,
+	y: 405,
+};
```

Comment:
What is the variable `START`? Maybe start a computer system or rockets.
Please rename this variable. So example `playerValue`

Reply:
done + add varible enemyCoordinate